src/components/Shared/LinePlot/LinePlotComponent.tsx

```diff
@@ -660,7 +661,8 @@ export class LinePlotComponent extends React.Component<LinePlotComponentProps> {
             // data part
             rows.push("# x\ty");
             const useScientificForm = plotName === "histogram" || this.props.tickTypeX === TickType.Scientific;
-            rows = rows.concat(this.props.data.map(o => (useScientificForm ? `${toExponential(o.x, 10)}\t${toExponential(o.y, 10)}` : `${o.x}\t${toExponential(o.y, 10)}`)));
+            const data = this.props.exportData?.length > 0 ? this.props.exportData : this.props.data;
```

Comment:
I'd prefer it if we used something like `fullResolutinoData` instead of `exportData`, as we may reuse this elsewhere.